### DISCUSSION TITLE
Autocomplete feature

### DISCUSSION THREAD

**Discussion by vigneshksaithal** [twbs/bootstrap#31903] [Ideas]
1. It is important because nowadays most websites have search capability and web apps also need search feature. As bootstrap is used by most developers they need this feature also in bootstrap instead of finding elsewhere.

**Comment by HashingTag** [comment]
I think your issue isn't clear. If you are asking about autocomplete related to HTML form elements, it's handled by the browser as far as I know.

**Comment by patrickhlauke** [comment]
a component that does autocomplete/find-ahead search will need all sorts of additional scripting behind the scenes (the full list of results available, the logic to then only show the relevant matches, etc). this does sound like something too specific/involved than what a general-purpose library/framework like BS can offer.

**Comment by vigneshksaithal** [comment]
> a component that does autocomplete/find-ahead search will need all sorts of additional scripting behind the scenes (the full list of results available, the logic to then only show the relevant matches, etc). this does sound like something too specific/involved than what a general-purpose library/framework like BS can offer.

I thought it as basic so I thought of raising request. But, from your inputs I got to know it is more advanced so shall I close this issue?

**Comment by patrickhlauke** [comment]
yeah, i think this may be a good candidate for some 3rd party plugin/extension, using bootstrap for styling, but probably not for bootstrap core itself. I'll move this to a discussion, in case others are interested in hashing out the topic further.
